upup/pkg/fi/cloudup/awstasks/elastic_ip.go

```diff
@@ -17,26 +17,31 @@ limitations under the License.
 package awstasks
 
 import (
-	"fmt"
-
+	//"fmt"
```

Comment:
won't fix - import diffs are functional not fashionable - result of developing - doesn't break anything